detekt.yml

```diff
@@ -84,7 +84,7 @@ formatting:
     active: true
     autoCorrect: true
   ExpressionBodySyntaxLineBreaks:
-      active: true
+      active: false
```

Comment:
I'm the waterparties here. The detek baseline will need to be regenerated and the long lines fixed if this is not enforced anymore :P